requirements.txt

```diff
@@ -1,2 +1,2 @@
-javasphinx==0.9.15
+myst-parser==0.17.2
 Sphinx==1.8.5
```

Comment:
now that the javadoc stuff is gone, is there still a reason for the outdated Sphinx version?